loadtest/control/simplecontroller/controller.go

```diff
@@ -71,15 +71,13 @@ func (c *SimpleController) Run() {
 		}
 	}()
 
+	i := 0
```

Comment:
I am wondering if it would be better to name this with a slightly more meaningful name since it might not be obvious at first glance what is the purpose of it.